### PR TITLE
Tomster player steps

### PR DESCRIPTION
**Pull request by marcoow**
This is the branch containing the individual steps for the Tomster player. This is not to be merged but we'll close this after approval. Please review each commit individually.

### CODE REVIEW

**Review by patsy-issa — COMMENTED** [review]
Overall lgtm 👍  one side note we could tackle in the ember-data step is to deal with the deprecation for albums/songs and comments:

```
store.serializerFor("comment") resolved the "-json-api" serializer via the deprecated `adapter.defaultSerializer` property.
```

**Comment by marcoow** [comment]
fixed the deprecation

File: tomster-player/app/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import Route from '@ember/routing/route';
+
+export default class IndexRoute extends Route {
+  model() {
```
[the remark this answers is not in the document]
Comment (marcoow) [reply tomster-player/app/routes/index.js]: We want it to be present for step 1 already so people don't have to add it before they understand what it even does.

**Comment by marcoow** [comment]
@patsy-issa this seems to be all good then – deployment to Netlify/Heroku works as well. I'm closing this then as it was only for review but is not intended to (ever) be merged as discussed.

**Pull request closed without merging**